README.md

```diff
@@ -52,7 +52,7 @@ Like `awesome-hyperterm`? Reach out to me and say *hi* on [Twitter](https://twit
 * [hyperseti](https://www.npmjs.com/package/hyperseti) - Dark - It's a theme based on [Seti](https://github.com/jesseweed/seti-ui), a subtle dark colored UI theme for Atom.
 * [hyperterm-oceanic-next](https://github.com/kasperlewau/hyperterm-oceanic-next) - Dark blue theme optimised for ES2015. Ported from [Oceanic Next](https://github.com/voronianski/oceanic-next-color-scheme).
 * [hyperterm-spacegray](https://www.npmjs.com/package/hyperterm-spacegray) – Port of the popular Spacegray theme, optimized for terminal usage.
-* [hyperterm-material](https://github.com/dperrera/hyperterm-material) - Dark - A material design inspired theme.
+* [hyperterm-material](https://www.npmjs.com/package/hyperterm-material) - Dark - A material design inspired theme.
```

Comment:
We definitely don't have to stick to the descriptions provided by the author. 😄 

Is there anything that you find aesthetically pleasing, or another reason why this theme is awesome? Would love to hear _your_ thoughts on it!